BlueShift-iOS-SDK/BlueShiftAppDelegate.m

```diff
@@ -378,7 +379,12 @@ - (void)handleRemoteNotification:(NSDictionary *)userInfo {
     if ([BlueshiftEventAnalyticsHelper isSilenPushNotificationPayload: userInfo]) {
         [[BlueShift sharedInstance] handleSilentPushNotification: userInfo forApplicationState: UIApplicationStateActive];
     } else {
+        NSString *pushUUID = [userInfo valueForKey:kInAppNotificationModalMessageUDIDKey];
         NSString *pushCategory = [[userInfo objectForKey: kNotificationAPSIdentifierKey] objectForKey: kNotificationCategoryIdentifierKey];
+        if ([pushCategory isEqualToString:kNotificationCategorySilentPushIdentifier] || [pushUUID isEqualToString:lastProcessedPushNotificationUUID]) {
+            [BlueshiftLog logInfo:@"Skipped processing notification due to following reasons" withDetails:@"1. The push notification is silent push notification 2. The push notification click is already processed." methodName:nil];
```

Comment:
I guess this would be a better message considering the condition here.
> Skipped processing notification due to one of the following reasons.

Reply:
sure, will change it.